src/content/docs/en/pages/guides/how-to-okta-saml/index.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Using Okta custom SAML app as an IdP for Real-Time Manager
+description: Hands-on guide on how to configure an Okta custom SAML application as an IdP for Azion Real-Time Manager.
+meta_tags: accounts, users authentication, SAML, okta, IdP
+namespace: documentation_sso_okta_saml
+permalink: /documentation/products/guides/sso-okta-saml/
+---
+
+The Okta Identity Provider (IdP) is a trusted entity that allows you to enable single sign-on to access other sites or services, such as Azion. Your users can continue using their corporate user identities without having to remember a specific password or enter credentials each time they access **Real-Time Manager (RTM)**. 
+
+:::caution[warning]
+Even if you're using an IdP, it’s necessary to register all users of the account on RTM as they are synced in both RTM and IdP. An inexistent user on RTM won’t be able to log in to it.
+:::
+
+When using an IdP, you have two protocol possibilities: Security Assertion Markup Language (SAML) and OpenID Connect (OIDC).
+
+To set up an Okta custom SAML app as your account IdP for RTM, follow the next steps.
+
+---
+
+## Configuring the SAML app on Okta Admin Console
+
+1. Log into the [Okta Admin Console](https://login.okta.com/).
+2. On the homepage, under **Applications**, select **Applications**. 
+3. Click **Create App Integration**.
+4. On the modal, select the option **SAML 2.0** and click **Next**.
+5. Name the application and configure optional settings. Example: `Azion SAML`.
+6. Click the **Next** button.
+7. In the **General** section, fill in the **Single sign-on URL** field with a URL. This will be adjusted after. Example: `https://sso.azion.com`.
+8. Repeat the previous step for the **Audience URI (SP Entity ID)** field.
+9. Scroll down to the **Attribute Statements** section and fill in the fields with the following data:
+    - **Name**: `email`
+    - **Name format**: Basic
+    - **Value**: `user.email`
+10. Click the **Next** button.
+11. Fill in the **Feedback** section as required.
+12. Click the **Finish** button.
+13. After you're redirected to your app details, click the **Sign On** tab.
+14. Under **Metadata details**, click **More details**.
+15. Copy the following metadata:
+    - **Sign on URL**, in the format `https://custom-name.okta.com/app/app-id/sso/saml`.
+    - **Issuer**, in the format `http://www.okta.com/app-id`.
+16. Scroll to the **SAML Signing Certificates** section and download the active certificate or generate a new one.
+
+---
+
+## Configuring the Identity Provider on RTM
+
+:::note
+You must be the **Account Owner** on RTM to perform these settings.
```

Comment:
Suggestion: add this link https://www.azion.com/en/documentation/products/accounts/teams-permissions/

Reply:
Adding in later review.